src/components/landing/tech-stack-cloud.tsx

```
@@ -138,26 +138,33 @@ function TechCard({
       )}
-      onMouseEnter={function () {
-        setHover(true)
-      }}
-      onMouseLeave={function () {
-        setHover(false)
-      }}
       {...props}
     >
       <motion.div
-        className="relative"
+        className="relative flex flex-col items-center gap-2"
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)